.travis.yml

```diff
@@ -4,8 +4,8 @@ services:
   - docker
 
 before_install:
-  - docker build -t yast-alternatives-image .
+  - docker build -t yast-update-alternatives-image .
```

Comment:
Oh, this was actually generated automatically by my magic script. :wink:

The repository was renamed from `update-alternatives` to `alternatives` and GitHub redirects the old name to the one automatically (just try accessing https://github.com/yast/yast-update-alternatives ...).

My script used the directory name for the image name and on my machine I have the checkout still in the original directory.